examples/example_plugin/example_plugin/ui/index.js

```diff
@@ -1,11 +1,15 @@
 export { default as ExampleAppFullWidthPageComponent } from "./FullWidthPage"
+export { default as ExampleAppModelRetrieveView } from "./ModelRetrieveView"
 
 const app_config = {
     detail_tabs: {},
     full_width_components: {
         "dcim:sites": ["ExampleAppFullWidthPageComponent"]
     },
-    view_overrides: {}
+    view_overrides: {},
+    routes_view_components: {
+        "examplemodel": "ExampleAppModelRetrieveView",
```

Comment:
Can you please document how this is used and what its structure is?

It seems weird to me that this is still associated to the `examplemodel` given the issue this is addressing - maybe a better example might be to use this to demonstrate how the example app can provide a "dashboard/home" view or a "config" view, either of which is entirely unrelated to the DB models?